### ISSUE TITLE
Using element web enables partially cross signing

### ISSUE THREAD

**Issue by odelcroi** [tchapgouv/tchap-web-v4#334]
### Steps to reproduce

- user did a key exchange with android application
- verification process with QR code showed
- user went through the verification process
- was succesfull

No proof it was caused by the verification process, there is a public cross signing key in memory now 
<img width="604" alt="Capture d’écran 2022-12-08 à 12 44 22" src="https://user-images.githubusercontent.com/4077729/206438450-806b0d1e-0c51-4ba3-82c8-5082dbc6f44f.png">

#### What did you expect?
Emoji process
No cross signing key

#### What happened instead?
QR code process possible
Cross signing key in memory now

### Operating system

MacOs

### Browser information

not sure

### URL for webapp

not sure

### Application version

not sure

### Will you send logs?

No

**Comment by jdauphant** [comment]
@odelcroi did the user part of ANSSI, DINUM or beta ?

**Comment by odelcroi** [comment]
yes

**Comment by odelcroi** [comment]
I reproduced it by connecting a brand new fresh account with a [app.element.com ](https://app.element.io/) client. After the device verification successful with a tchap web v4 session, I have this :  

![Capture d’écran 2022-12-08 à 18 19 20](https://user-images.githubusercontent.com/4077729/206520112-f8556199-9b5d-446d-8f98-4634935e57a1.png)


 
Note that I had to ignore the element web complaining that the account is not verified by closing all related windows and start the verification from the tchap client manually.

Note that during the verification, I hit this bug https://github.com/tchapgouv/tchap-web-v4/issues/318.
